packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -72,14 +72,18 @@ abstract contract FundingRateApplier is Testable, Lockable {
     /**
      * @notice Constructs the FundingRateApplier contract. Called by child contracts.
      * @param _finderAddress Finder used to discover financial-product-related contracts.
+     * @param _fundingRateRewardRate Reward rate to pay FundingRateStore.
      */
-    constructor(address _finderAddress) public {
+    constructor(address _finderAddress, FixedPoint.Unsigned memory _fundingRateRewardRate) public {
```

Comment:
1 new contructor param, used to call `setRewardRate()` on the FundingRateStore for this perpetual address